bin/check-redis-memory-percentage.rb

```diff
@@ -80,8 +80,12 @@ def run
     redis = Redis.new(options)
 
     redis_info = redis.info
-    memory_in_use = redis_info.fetch('used_memory').to_f.div(1024).to_f             # used memory in KB (KiloBytes)
-    total_memory = redis_info.fetch('maxmemory', system_memory).to_f.div(1024).to_f # max memory (if specified) in KB
+    max_memory = redis_info.fetch('maxmemory', 0).to_f
```

Comment:
The object type returned from `redis.info` are actually strings, which is probably why the casting to floats was being done in the first place. Even with all the casting of floats, the `.div(1024)`[ casts the result to an integer](https://ruby-doc.org/core-2.4.0/Numeric.html#method-i-div). If we really wanted to do floating point division, then we can use `.fdiv(1024)`. This could give slightly more accurate results if `maxmemory` is set to a value that is not a multiple of 1024. 

Since the return values from redis.info are number of bytes, then casting them to Integers, and using `fdiv(1024) ` might be more streamlined than the multiple instances of `.to_f`. This would revert the changes in #26. I'll go ahead and update the PR to include these changes